jujugui/static/gui/src/app/assets/css/inspector/_juju-inspector.scss

```diff
@@ -215,6 +74,22 @@
             &.wide {
                 width: 100%;
             }
+
+            &.confirm {
+                float: right;
+
+                &,
+                &:hover {
+                    background-color: $confirm-button-color;
```

Comment:
The confirm button should have a hover background of #ae3910 with a transition of background .3s